src/CONST.ts

```diff
@@ -570,6 +570,7 @@ const CONST = {
     GOOGLE_MEET_URL_ANDROID: 'https://meet.google.com',
     GOOGLE_DOC_IMAGE_LINK_MATCH: 'googleusercontent.com',
     IMAGE_BASE64_MATCH: 'base64',
+
     DEEPLINK_BASE_URL: 'new-expensify://',
```

Comment:
NAB: We prefer to not make unrelated changes on our PRs.

Reply:
reverted